superset/commands/report/execute.py

```diff
@@ -300,13 +300,16 @@ def _get_screenshots(self) -> list[bytes]:
         )
         user = security_manager.find_user(username)
 
+        max_width = app.config["ALERT_REPORTS_MAX_CUSTOM_SCREENSHOT_WIDTH"]
```

Comment:
@betodealmeida Maybe we should deprecate the `ALERT_REPORTS_MAX_CUSTOM_SCREENSHOT_WIDTH` feature flag to remove it in a major version. It seems that this feature should always be enabled in case alerts and reports are too.

Reply:
Oh, it's not a feature, just a configuration value.